deployment/ansible/roles/cac-tripplanner.transitfeed/tasks/main.yml

```diff
@@ -1,4 +1,3 @@
 ---
 - name: Install transitfeed
-  pip: name=pytz
```

Comment:
Okay, good catch on the second `pip`. For timezone data it's important to always use the latest, so not pegging the version seems appropriate. The transit feeds are currently also being validated as part of the download process with the feed fetcher library, but it seems useful for the app to enforce validation as well.